src/asmcnc/skavaUI/screen_dust_shoe_alarm.py

```diff
@@ -0,0 +1,349 @@
+                    height: dp(0.104166666667*app.height)
+                    width: dp(0.9*app.width)
+                    text_size: self.size
+                    size: self.parent.size
+                    x: self.parent.x + 80.0/800*app.width
+                    y: self.parent.y
+                    opacity: 1
+
+                Image:
+                    id: countdown_image
+                    source: "./asmcnc/skavaUI/img/countdown_big.png"
+                    x: self.parent.x
+                    y: self.parent.y
+                    height: self.parent.height
+                    allow_stretch: True
+                    opacity: 0
+
+
+        BoxLayout:
+            orientation: 'horizontal'
+            spacing: 0
+            size_hint: (None, None)
+            height: dp(0.270833333333*app.height)
+            width: dp(1.0*app.width)
+            padding:[0, 0, 0, dp(0.0208333333333)*app.height]
+   
+
+            Button:
+                font_size: str(0.01875 * app.width) + 'sp'
+                id: cancel_button
+                size_hint_x: 1
+                background_color: hex('#FFFFFF00')
+                on_press: root.cancel_stream()
+                disabled: True
+                BoxLayout:
+                    size: self.parent.size
+                    pos: self.parent.pos
+                    Image:
+                        source: "./asmcnc/skavaUI/img/cancel_from_pause.png"
+                        size: self.parent.width, self.parent.height
+                        allow_stretch: True
+            Button:
+                font_size: str(0.01875 * app.width) + 'sp'
+                id: resume_button
+                size_hint_x: 1
+                background_color: hex('#FFFFFF00')
+                on_press: root.resume_stream()
+                disabled: True
+                BoxLayout:
+                    size: self.parent.size
+                    pos: self.parent.pos
+                    Image:
+                        source: "./asmcnc/skavaUI/img/resume_from_pause.png"
+                        size: self.parent.width, self.parent.height
+                        allow_stretch: True 
+                
+
+"""
+)
+
+
+class DustShoeAlarmScreen(Screen):
+    poll_for_resume = None
+    return_to_screen = "home"
+    countdown_image = ObjectProperty()
+    spindle_raise_label = ObjectProperty()
+
+    def __init__(self, **kwargs):
+        super(DustShoeAlarmScreen, self).__init__(**kwargs)
+        self.sm = kwargs["screen_manager"]
+        self.m = kwargs["machine"]
+        self.jd = kwargs["job"]
+        self.db = kwargs["database"]
+        self.l = kwargs["localization"]
+        self.usm = App.get_running_app().user_settings_manager
+        self.header_label.text = self.l.get_bold("Interrupt bar pushed!")
+        self.spindle_raise_label.text = (
+            self.l.get_str("Preparing to resume, please wait") + "..."
+        )
+        self.anim_spindle_label = (
```

Comment:
Have these been tested on a pi? I experienced huge performance hits when using the Animation class.

Reply:
This is copied over from the door screen, and will be removed soon anyway

Reply:
Cool